lib/dialog/style.scss

```diff
@@ -1,45 +1,13 @@
-.dialogs {
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  z-index: $z-index-dialogs;
-}
-
-.dialogs-overlay {
-  content: ' ';
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background: rgba($white, 0.75);
-}
-
 .dialog {
-  display: flex;
-  position: absolute;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  padding: 10px;
-  overflow: auto;
-  -webkit-overflow-scrolling: touch;
-}
-
-.dialog-box {
-  flex: 1 1 auto;
-  margin: auto;
+  width: calc(100vw - 2rem);
   background: white;
   border-radius: $border-radius;
   border: 1px solid lighten($gray, 30%);
   box-shadow: 0 2px 4px 0 rgba($gray-darkest, 0.15);
```

Comment:
```suggestion
  box-shadow: 0 2px 4px 2px rgba($gray-darkest, 0.15);
```

I think if we add a 2px spread here it helps the dialog stand out a little bit more along the edges of the dialog, especially in the dark theme:
<img width="678" alt="screen shot 2018-10-23 at 1 49 42 pm" src="https://user-images.githubusercontent.com/789137/47389849-b39dc800-d6ca-11e8-9e21-e7af2696f3f5.png">

Reply:
Great idea!